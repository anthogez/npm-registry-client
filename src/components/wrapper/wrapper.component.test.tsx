import React from 'react';
import Adapter from 'enzyme-adapter-react-16';
import { shallow, configure } from 'enzyme';

import Wrapper from './';

configure({ adapter: new Adapter() });

describe('Trying render Wrapper Component and its children without crash.', () => {
	it('Check if Wrapper Component corrently Renders', () => {
		const wrapper = shallow(<Wrapper />);
		expect(wrapper).toMatchSnapshot();
	});

	it('Check if Header Component corrently renders', () => {
		const wrapper = shallow(<Wrapper />);
		expect(wrapper.find('Header')).toHaveLength(1);
	});

	it('Check if ContentResult Component corrently renders', () => {
		const wrapper = shallow(<Wrapper />);
		expect(wrapper.find('ContentResult')).toHaveLength(1);
	});
});
