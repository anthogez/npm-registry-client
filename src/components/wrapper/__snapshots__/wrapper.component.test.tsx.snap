// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Trying render Wrapper Component and its children without crash. Check if Wrapper Component corrently Renders 1`] = `ShallowWrapper {}`;
